source/libnormaliz/cone.cpp

```diff
@@ -906,6 +907,8 @@ void Cone<Integer>::process_multi_input_inner(map< InputType, vector< vector<Int
                 break;
             case Type::polyhedron:
                 inhom_input=true;
+                nr_cone_gen++;
+                break;
```

Comment:
This preserves semantics by making explicit what happens; but please verify that this is actually the intended behavior.